tests/ut_main.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import database
+import EScavange
+import gamemaker
+
+class TestDatabaseCreation(unittest.TestCase):
+  #Test that we can access the database. Should not test that the database DS is correct, that should be
+  #covered by the database tests.
+  
+  def setUp(self):
+    self.db = database.Database()
+    self.gameMaker = gamemaker.GameMaker()
+  
+  def accessDB(self):
+    pass
+    #stub for now : check that we can access the database
+    # How do i verify that the database is open???
+
+class TestLogin(unittest.TestCase):
+  #Test log in functionality. Will determine if username belongs to gamemaker or team and call correct method
+  
+  def setUp(self):
+    self.db = database.Database()
+    self.inputs = EScavange.main()
+    self.inputs.user = "Bob"            #Set the user inputs (so the test can be automatic, no user input necessary)
+    self.inputs.password = "password"
+        
+  def testLoginNoTeams(self):
+    self.assertEquals(EScavange.login(), "There is no such user. Please create a new team", "Login did not give the correct error message")
+
+  def testLoginTeam(self):
+    self.db.add_team(self.inputs.user,self.inputs.password) #What is the syntax to add a team to the DB?????????
+    self.assertEquals(EScavange.login(), "Currently logged in as Bob", "Login was not with the correct user")
```

Comment:
Same thing with this as the above comment. Should be calling login function on Team or GameMaker, otherwise how do we know what kind of user is logging in?